Remora.Discord.Commands/Attributes/DiscordDefaultDMPermissionAttribute.cs

```diff
@@ -0,0 +1,37 @@
+//
+//  DiscordDefaultDMPermissionAttribute.cs
+//
+//  Author:
+//       Jarl Gullberg <jarl.gullberg@gmail.com>
+//
+//  Copyright (c) 2017 Jarl Gullberg
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+using System;
+
+namespace Remora.Discord.Commands.Attributes;
+
+/// <summary>
+/// Marks a command group as being either executable or non-executable in a DM.
+/// </summary>
+[AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+public class DiscordDefaultDMPermissionAttribute : Attribute
+{
+    /// <summary>
+    /// Gets a value indicating whether the command group is executable in a DM.
```

Comment:
Might want to end this with "by default", since it can later be overridden.

Reply:
Overridden by?..

Reply:
User changes in the Discord UI, calls to the REST API, and so on.

Reply:
Whether a command is executable in DMs is purely up to the bot. Users can change guild-specific filtration via the UI/API.

Reply:
Ah, got it. The comment remains valid for the other attribute, though.